packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -169,6 +170,10 @@ class DataSource {
   /// detection with whatever is set here.
   final VideoFormat? formatHint;
 
+  /// HTTP headers used for the request to the [uri].
+  /// Only for [DataSourceType.network] videos.
+  Map<String, String>? httpHeaders;
```

Comment:
Non-nullable with empty default; see comment in other review.

Reply:
Fixed it